### PR TITLE
updated obsolete devtools info based on changelog

### PR DESCRIPTION
**Pull request by glebsts**
Getting started linked mobx-devtools, but it marked as obsolete in repo, also mobx changelog for v6 says few words about it and ives some recommendations, which I used to update the docs.

### CODE REVIEW

File: website/static/getting-started.html [hunk 1]
```diff
@@ -392,12 +392,30 @@ <h3>Asynchronous actions</h3>
...
+                            </li>
+                            <li>
+                                Spying on events can still be done with the
+                                <a href="https://github.com/mobxjs/mobx-devtools" target="_blank">MobX-react browser plugin</a>
+                                through the <a href="https://github.com/winterbe/mobx-logger" target="_blank">mobx-logger package</a>
+                                or manually by using the <code>spy</code> or <code>trace</code> utility from the mobx package.
+                            </li>
+                        </ul>
                         <p>
                             The <a href="https://github.com/mobxjs/mobx-react-devtools" target="_blank"><code>mobx-react-devtools</code></a> package
```
Comment (FredyC) [inline website/static/getting-started.html]: Please remove that duplicate `package` word, it's already in following brackets.

File: website/static/getting-started.html [hunk 2]
```diff
@@ -392,12 +392,30 @@ <h3>Asynchronous actions</h3>
...
+                                Visualizing re-rendering of components is now part of the standard
+                                <a href="https://reactjs.org/blog/2019/08/15/new-react-devtools.html" target="_blank">React devtools</a>
+                            </li>
+                            <li>
+                                The dependency tree of a component tree can be inspected by showing the state of the <code>useObserver</code>
+                                hook in the React devtools (older versions may display it as just an <code>Object</code>)
+                            </li>
+                            <li>
+                                Spying on events can still be done with the
+                                <a href="https://github.com/mobxjs/mobx-devtools" target="_blank">MobX-react browser plugin</a>
```
Comment (FredyC) [inline website/static/getting-started.html]:
```suggestion
                                <a href="https://github.com/mobxjs/mobx-devtools" target="_blank">browser plugin</a>
```

File: website/static/getting-started.html [hunk 3]
```diff
@@ -392,12 +392,30 @@ <h3>Asynchronous actions</h3>
...
+                                through the <a href="https://github.com/winterbe/mobx-logger" target="_blank">mobx-logger package</a>
```
Comment (FredyC) [inline website/static/getting-started.html]:
```suggestion
                                , through the <a href="https://github.com/winterbe/mobx-logger" target="_blank">mobx-logger package</a>
```

File: website/static/getting-started.html [hunk 4]
```diff
@@ -392,12 +392,30 @@ <h3>Asynchronous actions</h3>
...
+                                or manually by using the <code>spy</code> or <code>trace</code> utility from the mobx package.
```
Comment (FredyC) [inline website/static/getting-started.html]: Let's link it to docs for convenience
https://mobx.js.org/best/trace.html
https://mobx.js.org/refguide/spy.html

File: website/static/getting-started.html [hunk 5]
```diff
@@ -392,12 +392,30 @@ <h3>Asynchronous actions</h3>
                         </p>
 
                         <h3>DevTools</h3>
+                        <p>
+                            Following tools can be analyzed to analyze
```
Comment (FredyC) [inline website/static/getting-started.html]:
```suggestion
                            Following tools can be used to analyze
```

**Comment by FredyC** [comment]
Also would be lovely if you could contribute some detailed recipe to https://mobx-react.js.org/ which is more oriented on React. Personally, I don't use these tools at all, so I have no experience there.

**Comment by glebsts** [comment]
Thank you for all the comments! Patching..

> Also would be lovely if you could contribute some detailed recipe to https://mobx-react.js.org/ which is more oriented on React. Personally, I don't use these tools at all, so I have no experience there.

Have to skip that part - I'm new in this thingie and fellows recommended me mobx, so I just started reading it today.  Mb later when (and if) I get some experience.

**Pull request merged**